<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanel</string>
  </property>
  <widget class="QGroupBox" name="gpConfig">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="btnResetConfig">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>15s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labWinSize">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>31</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>窗大小(s)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gpExpType">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>56</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="rtnExpSwitch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Switch实验</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rtnExpShared">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shared实验</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rtnExpDirect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Direct实验</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rtnExpTunnel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tunnel实验</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gpTunnel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSetConfig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderWinSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="gpPlaceType">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>56</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="rtnSetTrace">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置Trace</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rtnSetRobot">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置Robot</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gpControlType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="rtnIdeal">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>理想模型</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rtnSimu">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>仿真模型</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cbContinuous">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连续控制</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbWithTTT">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用TTT预测</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gpOnTheRun">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>20s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labKusr">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>33</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kusr基准</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gpUserType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="rtnByPosition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>位置控制</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rtnByVelocity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>速度控制</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStartStop">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>116</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="labDecayTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>衰减时间(s)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sliderKusr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderQDecayTIme">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>60</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100倍</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1倍</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExitAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出所有</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExitMe">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出界面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
